### PR TITLE
cam6_2_031: Aerosol convective processes

### LINKED ISSUE #140: Bug in aerosol wet removal
The implementation of resuspension of aerosols from rain to largest mode in modal_aero_convproc.F90 is not correct.

### LINKED ISSUE #152: Update ChangeLog template to replace Bugzilla with Github
The `ChangeLog_template` file currently has the line:

`Bugs fixed (include bugzilla ID):`

However, bugzilla is no longer used, as the management of  bugs is now being handled by Github Issues.  Thus this text should be replaced with a request for the associated Github Issue (and possibly PR) numbers.

### PR DESCRIPTION
**Pull request by fvitt**
Refactor the implementation of resuspension of aerosols from rain to largest mode

Change to the calculation of convective vertical velocities of aerosols provided by Adam Herrington

Misc code clean up

Update to ChangeLog template

Fixes #140 
Fixes #152

### CODE REVIEW

File: src/chemistry/modal_aero/modal_aero_convproc.F90 [hunk 1]
```diff
@@ -2223,37 +2215,36 @@ subroutine ma_precpevap_convproc(                           &
          end if
       end do
  
-      ! Do resuspension of aerosols from rain to coarse mode (large particle) rather
+      ! Do resuspension of aerosols species from rain to coarse mode (large particle) rather
```
Comment (gold2718) [inline src/chemistry/modal_aero/modal_aero_convproc.F90:2218]: This seems like a correction. Is this correct or is it 'aerosol species'?

File: src/chemistry/modal_aero/modal_aero_convproc.F90 [hunk 2]
```diff
@@ -194,6 +198,9 @@ subroutine ma_convproc_init
    call addfld('DP_KCLDBASE', horiz_only, 'A', '1', &
                'Deep conv. cloudbase level index' )
 
+   call addfld ('WUP', (/ 'lev' /), 'A', 'm/s','ZM vertical velocity')
+   call addfld ('DPFRAC', (/ 'lev' /), 'A', 'unitless','ZM cloud fraction')
```
Comment (brian-eaton) [inline src/chemistry/modal_aero/modal_aero_convproc.F90:202]: 'unitless' is not a CF conforming unit (i.e., from udunits).  Should be '1'

I also don't find the required outfld calls.

[the remark this answers is not in the document]
Comment (fvitt) [reply src/chemistry/modal_aero/modal_aero_convproc.F90]: This diagnostics were not implemented correctly.  I removed these addfld calls.

**Comment by cacraigucar** [comment]
Requested rereview by @gold2718 as well for the ChangeLog_template mods.

File: doc/ChangeLog_template [hunk 3]
```diff
@@ -5,9 +5,7 @@ Originator(s):
 Date:
 One-line Summary:
```
Comment (nusbaume) [inline doc/ChangeLog_template:7]: I think this update is missing the line:

`Github PR URL:`

which should go between the `One-line Summary` and `Purpose of changes`.  Thanks!

Comment (fvitt) [reply doc/ChangeLog_template:8]: Does it look better now?

Comment (nusbaume) [reply doc/ChangeLog_template:8]: Yep, looks good.  Thanks!

**Pull request merged**